test/controllers/authorizeService.js

```diff
@@ -0,0 +1,184 @@
+var proxyquire = require('proxyquire').noCallThru();
+
+describe('Accounting Service', function () {
+
+     var DEFAULT_URL = 'http://example/path';
+     var DEFAULT_WSTOREHOST = 'localhost';
+     var DEFAULT_APIKEY = 'apiKey';
+
+     var getAuthorizeServiceController = function (accServiceSchema, config) {
+          return proxyquire('../../controllers/authorizeService', {
+               '../db/schemas/accountingService': accServiceSchema,
+               '../config': config
+          }).authorizeService;
+     };
+
+     var invalidRequest = function (handler, req, statusExpected, bodyExpected, done) {
+          var config = {
+               appHost: DEFAULT_WSTOREHOST
+          };
+
+          var accSerivceController = getAuthorizeServiceController({}, config);
+
+          res = jasmine.createSpyObj('res', ['status', 'json']);
+
+          res.status.and.returnValue(res);
+
+          accSerivceController[handler](req, res);
+
+          setTimeout(function () {
+
+               expect(res.status).toHaveBeenCalledWith(statusExpected);
+               expect(res.json).toHaveBeenCalledWith(bodyExpected);
+
+               done();
+
+          }, 100);
+     };
+
+     describe('Get api-key', function () {
+
+          it('should return 401 when the requester is not the WStore', function (done) {
+               var req = {
```

Comment:
You use this structure to many times. Create an auxiliary function to generate the `req` object.